app/models/scheduled_message.rb

```diff
@@ -2,4 +2,18 @@ class ScheduledMessage < ActiveRecord::Base
   validates :body, presence: true
   validates :days_after_start, presence: true
   validates :title, presence: true
+
+  before_save :escape_specific_characters
+
+  def body
+    self[:body].html_safe if self[:body]
```

Comment:
Totally fine on the style change, but in terms of validating the presence, this becomes an issue when trying to enter a new message.  There is no body yet on the object since it is being created at that point, so there is nothing to call html_safe on.  Since this method is overriding just accessing the property directly, it cannot assume that is instantiated in that case.